### PR TITLE
Update readme.md

### LINKED ISSUE #654: update our Techtonica/curriculum Readme Table of Contents
We have added a lot of content, and need help updating our Techtonica/curriculum Readme Table of Contents. https://github.com/Techtonica/curriculum#table-of-contents

Keep the currant format, but add any missing or outdated links in the "outline" column in the Techtonica Roadmap. This includes assessments and tutorial links, and any link in the "outline" column that isn't private access only.
https://docs.google.com/spreadsheets/d/1vIr3fiKa_jxzy7qtFsiE5hEJbiTXwc1LYxbfsHOYD6U/edit#gid=0

- The RoadMap is the source of truth.  If the order is different in the RoadMap, change the README to match.
- Note that the RoadMap is in two pages - The first half is in the 'archive' page, and the second is in the 2019 schedule page.
- No duplicates, please, even if there are duplicates on the Roadmap.
- Fix dead or incorrect links in Readme.
- Contact Alina for any questions about content.

### PR DESCRIPTION
**Pull request by ronika-das**
Fixes #654 

A couple of doubts:
Q1. Under which heading to put the following?
  a. [Twilio Quest Exercise Outline](/api/twilio-api.md)
  b. [Data Visualization](/data-visualization/data-visualization.md)
  c. [Assessment 7.9: Data Visualization](/data-visualization/data-viz-assessment.md)

Q2  I have added all assessments as well. Is that alright?

### CODE REVIEW

**Comment by alodahl** [comment]
Great question! Those should all go at the very end of the client-side section.  This looks excellent, thank you so much! Assessments should be in there unless it’s a replit link, yes. Also, isn’t there one more career week link?

File: README.md [hunk 1]
```diff
@@ -34,11 +34,11 @@ If you would like to contribute to the curriculum, please [sign up here](https:/
...
 1. [Welcome, Tips, and Rules](/welcome-tips-rules/tech-industry-tips.md)
 1. [How to Learn](/learning-to-learn/learning-to-learn.md)
-1. [Growth Mindset](/growth-mindset/YCLA_LessonPlan_v10.pdf)
 1. [Professionalism](/professionalism/professionalism.md)
-1. [Tech Industry Tips](/welcome-tips-rules/tech-industry-tips.md)
-1. [Roles in Tech](/roles-in-tech/roles-in-tech.md)
 1. [Ergonomics](/ergonomics/ergonomics.md)
+1. [Roles in Tech](/roles-in-tech/roles-in-tech.md)
+1. [Tech Industry Tips](/welcome-tips-rules/tech-industry-tips.md)
+1. [Growth Mindset](/growth-mindset/YCLA_LessonPlan_v10.pdf)
```
Comment (manaswinidas) [inline README.md]: All points start from number 1 :sweat_smile: Is it meant to be this way?

**Comment by manaswinidas** [comment]
Should we make another assessment session or something for the exercises? As all of the links are exercise and/or assessment links

File: README.md (hunk 1 above)
Comment (ronika-das) [reply README.md]: in the actual readme.md, the 1 gets inccremented
thats the syntax

**Comment by ronika-das** [comment]
@alodahl  i will do the necessary changes. I checked all the links,they are alright

**Comment by ronika-das** [comment]
the following are not being shown as links:
a. [Restaurant Recommender System with Javascript](/object-oriented-programming/restaurant recommender/RestaurantRecommender - part1.md)
b. [Restaurant Recommender System with Javascript - Part 2](/object-oriented-programming/restaurant recommender/RestaurantRecommender-part2.md

When I hover over the directory, it says path to empty directories

Tech Industry tips and Welcome ,Tips Rules were the same link. i removed the former

**Comment by alodahl** [comment]
@ronika-das, the files are there... see if you can fix them. There might be an extra space or something. https://github.com/Techtonica/curriculum/blob/master/object-oriented-programming/restaurant%20recommender/RestaurantRecommender-part2.md

**Comment by alodahl** [comment]
@manaswinidas no, everything should be in chronological order- ronika has it right.  Thanks for noticing, though.

**Comment by alodahl** [comment]
@ronika-das I found the welcome tips link! 
https://docs.google.com/document/d/1SWiUl0lYQ0sYDM8vQnVhJVOvT4MRA-9pDglKOgqTWak
Can you please put the tech tips link back?

**Comment by ronika-das** [comment]
@alodahl  I added the tech tips.
However unable to add the relative path after several attempts. I tried with spaces etc
1. /object-oriented-programming/restaurant recommender/RestaurantRecommender-part2.md
2.  /object-oriented-programming/restaurant recommender/RestaurantRecommender - part1.md

the object -oriented-programming is an empty directory(that is it has no files only folders). i believe that's why it is not working. this links might be able to provide some insight
a. https://github.blog/2014-08-19-folder-jumping/
b. https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository

**Comment by alodahl** [comment]
@ronika-das that's annoying, sorry about that!  Could you please:
- put the restaurant recommender/ folder in the javascript folder
-  delete object-oriented-programming/
- make sure any old links to /object-oriented-programming/restaurant recommender/ in the project now go to /javascript/restaurant recommender/  ?

Thanks, you're a champion!

**Comment by ronika-das** [comment]
on it!

**Comment by ronika-das** [comment]
i have deleted the oops folder and added the contents to javascript.
even then it wasn't working. Turns out that links should not have spaces between them. There was a space between Restaurant Recommender. I avoided this by using %20 instead of spaces. thought i'd share it with you :)

And i have not added this link from the sheet https://github.com/Techtonica/curriculum/blob/master/roles-in-tech/technical-writing.md
I think it was an example. i can add if u want

**Comment by alodahl** [comment]
@thank you Ronika!  Please do add technical-writing, as the file will be completed soon.

**Pull request merged**